web/src/js/connection.js

```diff
@@ -28,6 +30,8 @@ export default function Connection(url, dispatch) {
                 return dispatch(eventLogActions.updateLogEntries(message))
             case flowActions.UPDATE_FLOWS:
                 return dispatch(flowActions.updateFlows(message))
+            case settingsActions.WS_MSG_TYPE:
```

Comment:
It's a config entry but not an action type. So I think it's better to have a MSG_TYPE prefix for it.